discovery-provider/src/tasks/index_user_bank.py

```diff
@@ -147,6 +147,62 @@ def refresh_user_balances(session: Session, redis: Redis, accts=List[str]):
 ]
 
 
+def process_create_userbank_instruction(
+    session: Session,
+    instruction: TransactionMessageInstruction,
+    account_keys: List[str],
+    tx_sig: str,
+    timestamp: datetime.datetime,
+):
+    tx_data = instruction["data"]
+    parsed_token_data = parse_create_token_data(tx_data)
+    eth_addr = parsed_token_data["eth_address"]
+    decoded = base58.b58decode(tx_data)[1:]
+    public_key_bytes = decoded[:20]
+    mint_address = account_keys[
+        get_account_index(instruction, CREATE_MINT_ACCOUNT_INDEX)
+    ]
+    _, derived_address = get_address_pair(
+        PublicKey(mint_address), public_key_bytes, USER_BANK_KEY, SPL_TOKEN_ID_PK
+    )
+    bank_acct = str(derived_address[0])
+    try:
+        # Confirm expected address is present in transaction
+        bank_acct_index = account_keys.index(bank_acct)
+        if bank_acct_index:
```

Comment:
Worth logging or something if there's no bank_acct_index? (I know you just extracted)